app/controllers/room_guests_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RoomGuestsController < ApplicationController
+  def create
+    guest = Guest.find(params[:id])
+    room = Room.find(params[:room_id])
+    RoomGuest.create!(guest_id: guest.id, room_id: room.id)
```

Comment:
We are not expecting much error handling in mid mods, but keep in mind that in general, it's best to use `create!` only in tests or seeds, and in controllers just use `create` so that users are not exposed to unexpected exceptions being thrown.